### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by Amber1990Zhang**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,71 +1,103 @@
...
+
+Run the following command to build Nebula Graph Console.
+
+```bash
 $ cd nebula-console
 $ make
 ```
 
-# Usage
+## Connect Nebula Graph Console
```
Comment (randomJoe211) [inline README.md:29]: Connect to Nebula Graph?

File: README.md [hunk 2]
```diff
@@ -1,71 +1,103 @@
...
+
+Bye root!
 ```
 
-# Feature
+## FAQ
+
+TODO [This section should include connection-related FAQs. When users meet a problem on connection, they should first come here for a solution. If the content gets stacked, remove this section to a single topic.]
+
+
```
Comment (randomJoe211) [inline README.md:97]: Remove this section?

**Review by randomJoe211 — DISMISSED** [review]

File: README.md [hunk 3]
```diff
@@ -1,71 +1,96 @@
...
+Run the following command to examine if Go is installed on your machine.
+
+```bash
+$ go version
+```
+
+Run the following command to build Nebula Graph Console.
+
+```bash
```
Comment (whitewum) [inline README.md:24]: better give a command of `git clone`

Comment (whitewum) [reply README.md:24]: souce code

**Review by whitewum — DISMISSED** [review]

File: README.md [hunk 4]
```diff
@@ -1,71 +1,96 @@
...
+This repository contains the Nebula Graph Console for Nebula Graph 2.0.
 
-# Build
+## Features
 
-Install golang by https://golang.org/doc/install, then build with following commands:
+- Supports interactive and non-interactive mode.
+- Supports viewing the history statements.
+- Supports autocompletion.
+- Supports Multiple OS and arch (We recommend linux/amd64).
```
Comment (oldLady344) [inline README.md:10]: why "Multiple" is capitalized?

arch ==> architecture?

linux ==> Linux? amd64 ==> AMD64

Linux and AMD64 are recommended?

File: README.md [hunk 5]
```diff
@@ -1,71 +1,96 @@
...
 ```
 
+| Option       | Description                                                                                                                                                                   |
+| ------------ | ----------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+|`-h`           | Shows the help menu.                                                                                                                                                           |
+| `-addr`        | Sets the IP address of the graphd service. The default address is 127.0.0.1.                                                                                                   |
+| `-port`        | Sets the port number of the graphd service. The default port number is 3699.                                                                                                   |
+| `-u/-user`     | Sets the username of your Nebula Graph account. The default username is root.                                                                                                  |
```
Comment (oldLady344) [inline README.md:43]: \`root\`

File: README.md [hunk 6]
```diff
@@ -1,71 +1,96 @@
...
+| `-p/-password` | Sets the password of your Nebula Graph account. The default password for the root user is nebula.                                                                              |
```
Comment (oldLady344) [inline README.md:44]: \`root\` 

\`nebula\`

File: README.md [hunk 7]
```diff
@@ -1,71 +1,96 @@
...
+| `-t/-timeout`  | Sets an integer-type timeout threshold for the connection. The unit is second. The default value is 120.                                                                        |
+| `-e/-eval`     | Sets a string-type nGQL statement. The nGQL statement is executed once the connection succeeds. The connection stops after the result is returned..             |
```
Comment (oldLady344) [inline README.md:46]: extra period

File: README.md [hunk 8]
```diff
@@ -1,71 +1,96 @@
...
+| `-f/-file`     | Sets the path of an nGQL file. The nGQL statements in the file are executed once the connection succeeds. You'll get the return messages and the connection stops then. |
```
Comment (oldLady344) [inline README.md:47]: You'll get the return messages and the connection stops then. [This means when the result is returned the connection will stop? ]

File: README.md [hunk 9]
```diff
@@ -1,71 +1,96 @@
...
+
 Check options for `./nebula-console -h`, try `./nebula-console` in interactive mode directly.
 And try `./nebula-console -e 'show hosts'` for the direct script mode.
 And try `./nebula-console -f demo.nGQL` for the script file mode.
```
Comment (oldLady344) [inline README.md:51]: It means nebula-console has three modes? if yes, a sentence introducing these modes would be welcome. And then, you can start to describe how to try these modes.

File: README.md (hunk 4 above)
Comment (whitewum) [reply README.md:10]: Linux/AMD64

**Review by randomJoe211 — APPROVED** [review]
LGTM

**Pull request merged**